third-party/cmake/BuildLuajit.cmake

```diff
@@ -30,13 +30,14 @@ function(BuildLuajit)
     CONFIGURE_COMMAND "${_luajit_CONFIGURE_COMMAND}"
     BUILD_IN_SOURCE 1
     BUILD_COMMAND "${_luajit_BUILD_COMMAND}"
-    INSTALL_COMMAND "${_luajit_INSTALL_COMMAND}")
+    INSTALL_COMMAND "${_luajit_INSTALL_COMMAND}"
+    DEPENDS "${_luajit_DEPENDS}")
 
   # Create symlink for development version manually.
   if(UNIX)
     add_custom_command(
       TARGET ${_luajit_TARGET}
-      COMMAND ${CMAKE_COMMAND} -E create_symlink luajit-2.1.0-beta3 ${DEPS_BIN_DIR}/luajit)
+      COMMAND ${CMAKE_COMMAND} -E create_symlink luajit-2.1.0-beta3 ${DEPS_BIN_DIR}/${_luajit_TARGET})
```

Comment:
This seems to make my build woes go away. I don't understand why, but it does.

Reply:
lolwat? Aren't they the same string?

Reply:
Not exactly. I build intermediate luajit targets [here](https://github.com/neovim/neovim/blob/fd2329a80be0ff405e058cbed0d5ebc4201e3a73/third-party/cmake/BuildLuajit.cmake#L91-L96):
```cmake
    BuildLuaJit(TARGET "luajit-${ARCH}"
        BUILD_COMMAND ${BUILDCMD_UNIX}
        CC=${LUAJIT_C_COMPILER} STATIC_CC=${STATIC_CC}
        DYNAMIC_CC=${DYNAMIC_CC} TARGET_LD=${TARGET_LD}
        PREFIX=${DEPS_INSTALL_DIR}
        ${DEPLOYMENT_TARGET})
```